### PR TITLE
sort descriptions

### PR DESCRIPTION
**Pull request by FernandoMiguel**
### What does this PR do?
sorted the description, since it was all over the place

### More

- [ ] Yes, I have tested the PR using my local account setup  (Provide any test evidence report under Additional Notes)
- [ ] Yes, I have added a new example under [examples](https://github.com/aws-ia/terraform-aws-eks-blueprints/tree/main/examples) to support my PR
- [ ] Yes, I have created another PR for add-ons under [add-ons](https://github.com/aws-samples/eks-blueprints-add-ons) repo (if applicable)
- [ ] Yes, I have updated the [docs](https://github.com/aws-ia/terraform-aws-eks-blueprints/tree/main/docs) for this feature
- [ ] Yes, I ran `pre-commit run -a` with this PR

**Note**: Not all the PRs required examples and docs except a new pattern or add-on added.

### For Moderators
- [ ] E2E Test successfully complete before merge?

### CODE REVIEW

**Comment by vara-bonthu** [comment]
We have noticed that breaking change hence i raised a PR #610 to cover this issue.

**Comment by vara-bonthu** [comment]
> @vara-bonthu cool cool. shall i change mine to just sort the remaining descriptions?

Yes, you can keep the sorting. I am wondering if the sorting can be done with  pre-commit? Nice to have this in the workflows

File: modules/kubernetes-addons/variables.tf [hunk 1]
```diff
@@ -1,5 +1,6 @@
 variable "eks_cluster_id" {
   description = "EKS Cluster Id"
+  default     = ""
```
Comment (vara-bonthu) [inline modules/kubernetes-addons/variables.tf:3]: Don't set the empty value. This is a mandatory value for this add-on

Comment (FernandoMiguel) [reply modules/kubernetes-addons/variables.tf:3]: > I am wondering if the sorting can be done with pre-commit? Nice to have this in the workflows

the problems will be the more complex maps

Comment (FernandoMiguel) [reply modules/kubernetes-addons/variables.tf:3]: > Don't set the empty value. This is a mandatory value for this add-on

your PR should have address this... i'll drop all of my changes

File: modules/kubernetes-addons/variables.tf [hunk 2]
```diff
@@ -22,43 +22,43 @@ variable "auto_scaling_group_names" {
...
 variable "eks_cluster_endpoint" {
-  type        = string
   description = "Endpoint for your Kubernetes API server"
+  type        = string
 }
 
 variable "eks_cluster_version" {
-  type        = string
   description = "The Kubernetes version for the cluster"
+  type        = string
```
Comment (vara-bonthu) [inline modules/kubernetes-addons/variables.tf:54]: I am setting these three default values to `null` in my PR. Once its merged then you can rebase from Main

**Review by vara-bonthu — COMMENTED** [review]
LGTM! One comment on the PR

File: modules/kubernetes-addons/variables.tf (hunk 2 above)
Comment (FernandoMiguel) [reply modules/kubernetes-addons/variables.tf:54]: if you want, i can merge mine into yours instead of main

Comment (askulkarni2) [reply modules/kubernetes-addons/variables.tf:54]: Probably better to merge this PR first.

**Comment by FernandoMiguel** [comment]
conflicts fixed @vara-bonthu @askulkarni2

**Comment by FernandoMiguel** [comment]
@bryantbiggs the code in this PR is not in 616

**Review by bryantbiggs — APPROVED** [review]
Thank you @FernandoMiguel

**Pull request merged**